ch2driver/pytpcc/drivers/nestcollectionsdriver.py

```diff
@@ -734,17 +804,21 @@ def __init__(self, ddl, clientId, TAFlag="T",
             return
 
         if TAFlag == "T":
-            for txn in TXN_QUERIES:
-                for query in TXN_QUERIES[txn]:
+            if self.schema == constants.CH2_DRIVER_SCHEMA["CH2"]:
+                self.txnQueries = TXN_QUERIES
+            else:
+                self.txnQueries = CH2PP_TXN_QUERIES
+            for txn in self.txnQueries:
+                for query in self.txnQueries[txn]:
```

Comment:
nitpick: this can be written like so:
```
for txn, queries in self.txnQueries.items():
    for query, statement in queries.items():
        ...
```
And then instead of having to write `self.txnQueries[txn][query]` within the loops, you can just use `statement`.

Reply:
Done